### PR TITLE
DOC: Document missing_kwds in `explore`

### PR DESCRIPTION
**Pull request by ChristofKaufmann**
The documentation for `missing_kwds` in `explore` was missing.

### CODE REVIEW

File: geopandas/explore.py [hunk 1]
```diff
@@ -204,6 +204,13 @@ def my_colormap(value):  # scalar value defined in 'column'
     highlight_kwds : dict (default {})
         Style to be passed to folium highlight_function. Uses the same keywords
         as ``style_kwds``. When empty, defaults to ``{"fillOpacity": 0.75}``.
+    missing_kwds : dict (default {})
+        Additional style for missing values (NaN):
```
Comment (martinfleis) [inline geopandas/explore.py:208]:
```suggestion
        Additional style for missing values:
```

Not only NaN but also None, depending on the dtype.

File: geopandas/explore.py [hunk 2]
```diff
@@ -204,6 +204,13 @@ def my_colormap(value):  # scalar value defined in 'column'
...
+
+        color : str
+            Color of NaN values. Defaults to ``None``, which uses Folium's default.
+        label : str (default "NaN")
+            Legend entry for missing values.
```
Comment (martinfleis) [inline geopandas/explore.py:213]:
```suggestion
        Uses the same keywords as ``style_kwds``. Additionally, allows specification
        of the legend label.

        label : str (default "NaN")
            Legend entry for missing values.
```

It is not just color but any supported style.

**Review by martinfleis — COMMENTED** [review]
Thanks! Some comments inline.

File: geopandas/explore.py (hunk 2 above)
Comment (ChristofKaufmann) [reply geopandas/explore.py:213]: Thank you for your feedback, but I think it does not support the same keywords as `style_kwds`. All uses of `missing_kwds` I could find are:

- [`nan_color = missing_kwds.pop("color", None)`](https://github.com/geopandas/geopandas/blob/main/geopandas/explore.py#L554)
- `categories.append(missing_kwds.pop("label", "NaN"))` in [L682](https://github.com/geopandas/geopandas/blob/main/geopandas/explore.py#L682) and [L721](https://github.com/geopandas/geopandas/blob/main/geopandas/explore.py#L721)
- [`    m, "", [missing_kwds.pop("label", "NaN")], [nan_color]`](https://github.com/geopandas/geopandas/blob/main/geopandas/explore.py#L757)

Could you have a look again?

File: geopandas/explore.py [hunk 3]
```diff
@@ -204,6 +204,13 @@ def my_colormap(value):  # scalar value defined in 'column'
     highlight_kwds : dict (default {})
         Style to be passed to folium highlight_function. Uses the same keywords
         as ``style_kwds``. When empty, defaults to ``{"fillOpacity": 0.75}``.
+    missing_kwds : dict (default {})
+        Additional style for missing values:
+
+        color : str
+            Color of NaN values. Defaults to ``None``, which uses Folium's default.
```
Comment (martinfleis) [inline geopandas/explore.py:211]:
```suggestion
            Color of missing values. Defaults to ``None``, which uses Folium's default.
```

**Review by martinfleis — APPROVED** [review]
Thanks!

**Pull request merged**